src/routes/concepts/stores.mdx

```diff
@@ -238,157 +232,145 @@ setStore("users", (otherUsers) => [
+It determines the differences between the existing data and the incoming data and modifies only the _changed_ values, avoiding unnecessary updates.
+In this example, only `'koala'`, the new addition, will cause an update:
 
 ```jsx
 const { createStore, reconcile } from "solid-js/stores"
 
-const [data, setData] = createStore({
+const [store, setStore] = createStore({
 	animals: ['cat', 'dog', 'bird', 'gorilla']
 })
 
-const newData = getNewData() // eg. contains ['cat', 'dog', 'bird', 'gorilla', 'koala']
-setData('animals', reconcile(newData))
-
+const newData = {
+	animals: ['cat', 'dog', 'bird', 'gorilla', 'koala']
+}
+setStore('animals', reconcile(newData))
 ```
 
-In this example, the store will look for the differences between the existing and incoming data sets.
-Consequently, only `'koala'` - the new edition - will cause an update.
+### Extracting data with `unwrap`
 
-### Extracting raw data with `unwrap`
+The `unwrap` utility lets you transform a store into a standard [object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object), which is useful when you need the data in a non-reactive situation.
+It gives you a snapshot of the current state without the overhead associated with reactivity.
 
-When there is a need for dealing with data outside of a reactive context, the `unwrap` utility offers a way to transform a store to a standard [object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object).
-This conversion serves several important purposes.
-
-Firstly, it provides a snapshot of the current state without the processing overhead associated with reactivity.
-This can be useful in situations where an unaltered, non-reactive view of the data is needed.
-Additionally, `unwrap` provides a means to interface with third-party libraries or tools that anticipate regular JavaScript objects.
-This utility acts as a bridge to facilitate smooth integrations with external components and simplifies the incorporation of stores into various applications and workflows.
+This is useful in situations where you need an unaltered, non-reactive view of the data, for example, to interface with third-party libraries that anticipate regular JavaScript objects.
+It facilitates integrating external components and simplifies incorporating stores into applications and workflows outside of Solid.
 
 ```jsx
-import { createStore, unwrap } from "solid-js/store"
+import { createStore, unwrap } from "solid-js/store";
 
-const [data, setData] = createStore({
+const [store, setStore] = createStore({
 	animals: ["cat", "dog", "bird", "gorilla"],
-})
+});
 
-const rawData = unwrap(data)
+const rawData = unwrap(store);
 ```
 
-To learn more about how to use Stores in practice, visit the [guide on complex state management](/guides/complex-state-management).
+To learn more about how to use Stores in practice, visit the [complex state management guide](/guides/complex-state-management).
```

Comment:
I read this page too and a lot of the content there is duplicative of the content on this page. I think some of the examples there might also be a little misleading because they demonstrate synchronizing state instead of deriving it, which as I understand it is a Solid antipattern.

If this PR goes OK, I'd love to try something similar (maybe less ambitious) for that page.